base/applications/notepad/lang/pt-PT.rc

```diff
@@ -1,4 +1,4 @@
-LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
```

Comment:
I don't understand that change. Isn't it our goal to always have one SUBLANG_NEUTRAL and then
e.g. for pt-**BR** the line LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE_BRAZILIAN?

`LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE_BRAZILIAN` is what is current written in notepads pt-BR. So after your change there wouldn't be any SUBLANG_NEUTRAL left for any pt-\*. Isn't that wrong?

**What does your change fix actually?**

For the record for regedit we do have
in pt-PT.rc:
`LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL`

in pt-BR.rc:
`LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE_BRAZILIAN`

Why do you want to do it differently foir each module?

Reply:
That was my understanding of rule two at https://reactos.org/wiki/Sublang and that was what I did for all pt-PT translations. However, someone commented in a review that it should be changed to SUBLANG_PORTUGUESE if there was a translation into pt-BR. Although I wasn't very convinced, I started using SUBLANG_PORTUGUSE in case there was a pt-BR.
Now I'm sure how I should proceed. Thanks.

Reply:
That someone very likely commented that on some PR of yours to some module, where pt-BR had the 
`LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL`
set. Which isn't the case here.
We do have to decide which of both should be the neutral one, when both do exist.

From your linked wiki-article you do violate the following rule with your current PR
`If the primary language has more than one sublanguage and we have no general translation for this primary language yet, use SUBLANG_NEUTRAL for the general translation.`
You do violate that rule for your current notepad-PR. And you didn't violate that rule for your last regedit PR.

**This conversation is not resolved, please do not mark it as resolved.**

Reply:
he changed back to NEUTRAL now. This is ok. Resolved now.